tests/modules/prinseqplusplus/test.yml

```diff
@@ -0,0 +1,29 @@
+- name: prinseqplusplus test_prinseqplusplus_single_end
+  command: nextflow run tests/modules/prinseqplusplus -entry test_prinseqplusplus_single_end -c tests/config/nextflow.config
+  tags:
+    - prinseqplusplus
+  files:
+    - path: output/prinseqplusplus/test.log
+      contains:
+        - "reads removed by -lc_entropy"
+    - path: output/prinseqplusplus/test_bad_out.fastq.gz
+    - path: output/prinseqplusplus/test_good_out.fastq.gz
+    - path: output/prinseqplusplus/versions.yml
```

Comment:
I think we said we wouldn't check the md5sum anymore, too much hassle for version updates https://github.com/nf-core/modules/pull/1459/commits/b90e590dff5d96c382ac2c50cc47f8f072ace9dd

Reply:
Is that documented somewhere and I missed it

Reply:
Can't dinf it. @maxulysse , I saw it in your PR, any docs on this?